src/bitcoin-cli.cpp

```diff
@@ -632,25 +659,28 @@ class NetinfoRequestHandler : public BaseRequestHandler
     }
 
     const std::string m_help_doc{
-        "-netinfo level|\"help\" \n\n"
+        "-netinfo level|outonly|help\n\n"
```

Comment:
nit: `level|outonly|help` reads as "exactly one of `level`, `outonly` or `help`". I guess it should be

```
-netinfo (level [outonly]) | help
```
or two lines:

```
        "-netinfo \"help\"\n"
        "-netinfo level [outonly]\n\n"
```

Reply:
Thanks! Done.